lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -39,7 +39,7 @@ import {
 } from "@azure/msal-common/node";
 import {
     Configuration,
-    buildAppConfiguration,
+    _buildAppConfiguration,
```

Comment:
Why `_`? We are limiting `_` usage to JS generated code.
Refer: [naming style](https://google.github.io/styleguide/tsguide.html#naming-style) and [generated code](https://google.github.io/styleguide/tsguide.html#generated-code)